src/utils/validators/index.ts

```diff
@@ -23,3 +23,13 @@ export const validatePercentage = (percent: any) => {
 
 export const validateSlack = (v: string): boolean =>
   v.match("(#|@).+") !== null;
+
+/**
+ *  validatePatchId tests if a provided id is a mongo objectId indicating that it likely belongs to a patch and not a version
```

Comment:
This comment is formatted oddly, do we get to blame autopilot?

Reply:
This comment is using [jsdoc](https://jsdoc.app/) syntax 
![image](https://user-images.githubusercontent.com/4605522/127235625-274ae107-335d-4076-b4a4-dce070187bba.png)